en_US/deploy/cluster/security.md

```diff
@@ -1,55 +1,58 @@
 # Cluster security
 
-When it comes to the security of the EMQX cluster, there two primary
-aspects to consider.
+Security is the cornerstone of all IoT applications and platforms. EMQX adopts multiple protection mechanisms to ensure data and privacy security, for example, EMQX supports multiple [authentication](../../access-control/authn/authn.md) and [authorization](../../access-control/authz/authz.md) mechanisms on the node level. It also leverages the features of port mapping and TLS encryption to ensure the security of client data transmission, message communication between cluster nodes, and enterprise system integrations.
```

Comment:
The scope is too large and the recommendation is to focus on cluster security

Reply:
please have another look @wivwiv